atd-vzd/metadata/databases/default/tables/public_crashes.yaml

```diff
@@ -92,7 +92,7 @@ select_permissions:
         - crash_timestamp
         - created_at
         - created_by
-        - engineering_area
+        - engineering_area_id
```

Comment:
decided to rename this column to `engineering_area_id` to differentiate between id and the actual name of the eng area